fms2_io/netcdf_io.F90

```diff
@@ -245,9 +246,24 @@ module netcdf_io_mod
 contains
 
 !> @brief Accepts the namelist fms2_io_nml variables relevant to netcdf_io_mod
-subroutine netcdf_io_init (chksz)
+subroutine netcdf_io_init (chksz, netcdf_default_format)
 integer, intent(in) :: chksz
+character (len = 10), intent(in) :: netcdf_default_format
+
  fms2_ncchksz = chksz
+ if (string_compare(netcdf_default_format, "64bit", .true.)) then
```

Comment:
Does `string_compare` trim the strings?  I think you should probably use 
```fortran
if (string_compare(trim(netcdf_default_format), "64bit", .true.)) then
```
to be safe

Reply:
`string_compare` trims the strings for you. 
https://github.com/NOAA-GFDL/FMS/blob/d34bb9fa1107c5cc08a552eefd7b127a22c3a49d/fms2_io/fms_io_utils.F90#L228